src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -31,16 +31,21 @@ public class Win32 : IProgram
         public bool Valid { get; set; }
         public bool Enabled { get; set; }
         public bool hasArguments { get; set; } = false;
+        public string Arguments { get; set; } = String.Empty;
         public string Location => ParentDirectory;
-        public string AppType { get; set; }
+        public uint AppType { get; set; }
 
         private const string ShortcutExtension = "lnk";
         private const string ApplicationReferenceExtension = "appref-ms";
         private const string ExeExtension = "exe";
         private const string InternetShortcutExtension = "url";
-
-        private const string InternetShortcutApplication = "Internet shortcut application";
-        private const string Win32Application = "Win32 application";
+
+        private const string proxyWebApp = "_proxy.exe";
+        private const string appIdArgument = "--app-id";
+
+        private const uint WEB_APPLICATION = 0;

```

Comment:
@alekhyareddy28 .  One nit comment can this be an enum?  I would find that slightly more readable.